.github/workflows/pkgci_test_amd_mi355.yml

```
@@ -0,0 +1,102 @@
+# Copyright 2025 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: PkgCI Test AMD MI355
+permissions:
+  contents: read
+on:
+  workflow_call:
+    inputs:
+      artifact_run_id:
+        type: string
+        default: ""
+  workflow_dispatch:
+    inputs:
+      artifact_run_id:
+        type: string
+        default: ""
+
+jobs:
+  test_amd_mi355:
+    name: "test_mi355"
+    strategy:
+      fail-fast: false
```

suggested changes for consistency with other workflows
```suggestion
  test_mi355:
```